web3j/src/main/java/com/vmware/ethereum/service/SecureTokenApi.java

```diff
@@ -62,6 +62,7 @@
   private final String senderAddress;
   private SecurityToken token;
   private String contractAddress;
+  private int recipientIndex = 0;
```

Comment:
Let's be more declarative rather than imperative. Replace this with Iterator.
``private Iterator<String> recipients;``

Import:
``import static com.google.common.collect.Iterators.cycle;``

In the init()
`` recipients = cycle(config.getRecipients());``

Reply:
done